pages/glossary/mechanism.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Mechanism to achieve an objective
+key: mechanism-to-achieve-an-objective
+unambiguous: true
+objective: false
+---
+
+A mechanism to achieve an objective is a set of [HTML elements][] and [scripting][] in an [HTML document][] that allow an end-user to achieve an objective. 
```

Comment:
Can't find any rules with this "play button" definition :-(

Reply:
> Do they have to read every line of code of the web page and somehow understand all of it before they can decide there is no mechanism?

I would say so. Proving something doesn't exist always involves checking every possible spot it could exist. I don't see a problem with this. This is what the SC asks of us.